lerna.json

```diff
@@ -3,7 +3,8 @@
   "version": "independent",
   "command": {
     "publish": {
-      "message": "chore: release"
+      "message": "chore: release",
+      "allowBranch": ["master", "feature/"]
```

Comment:
@mattrothenberg, this is what will allow us to publish canary builds for branches prefixed with `feature/`.